Purchases/RCPurchaserInfo+Protected.h

```diff
@@ -18,8 +18,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSDictionary<NSString *, NSObject *> *purchaseDateByEntitlement;
 @property (nonatomic, readonly) NSSet<NSString *> *nonConsumablePurchases;
 @property (nonatomic, readonly) NSString  * _Nullable originalApplicationVersion;
+@property(readonly) NSString * _Nullable schemaVersion;
 
 - (NSDictionary * _Nonnull)JSONObject;
++ (NSString *)schemaVersion;
```

Comment:
So is this a class variable or an instance variable? `[self schemaVariable]` could call either depending. It's probably better to give this a different name than to shadow it.